apps/mobile/src/app/(home)/settings/help/index.tsx

```diff
@@ -23,10 +26,12 @@ export default function SettingsHelpScreen() {
           })}
           caption={t({
             id: 'help.support.cell_caption',
-            message: 'Placeholder',
+            message: 'Contact our support team',
```

Comment:
Yeah lets think about it and see 👍 with your recent fixes it seems to be working well and publishing OTA. It could be we just need to document it a bit / have a stricter process.

I think it would still be nice to have an emergency fallback text though